src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.service.ReportService;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private StorageDao storageDao;
+
+    public ReportServiceImpl(StorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
```

Comment:
```suggestion
```

report services shouldn't know about dao. pass map with fruits in method parameter instead